osclib/request_splitter.py

```diff
@@ -140,6 +140,8 @@ def supplement(self, request):
         request.set('postponed', 'False')
 
     def ring_get(self, target_package):
+        if self.api.conlyadi:
+            return None
```

Comment:
This does not handle:

https://github.com/openSUSE/openSUSE-release-tools/blob/43558590825d2a9ecfdadde999a32c3aa82644c4/osclib/request_splitter.py#L130-L132

which will likely need `!self.api.conlyadi and ...`.